Makefile

```diff
@@ -31,6 +16,7 @@ help:
 	@echo "    tidy:            tidy go mod"
 	@echo "    lint:            run golangci-lint"
 	@echo "    build:           run all builds"
+	@echo "    integration-test: run integration tests"
```

Comment:
Let's add a "test" target that runs all tests?

Reply:
Done.